src/app/pages/TokenDashboardPage/hook.ts

```diff
@@ -34,12 +35,9 @@ export const WANTED_EVM_LOG_EVENTS_FOR_LISTING_TOKEN_TRANSFERS: string[] = ['Tra
 
 export const useTokenTransfers = (scope: SearchScope, address: string) => {
   const { network, layer } = scope
-  const pagination = useClientSizePagination({
+  const pagination = useComprehensiveSearchParamsPagination<RuntimeEvent, RuntimeEventList>({
     paramName: 'page',
-    clientPageSize: NUMBER_OF_ITEMS_ON_SEPARATE_PAGE,
-    serverPageSize: 1000,
-    filter: (event: RuntimeEvent) =>
-      !!event.evm_log_name && WANTED_EVM_LOG_EVENTS_FOR_LISTING_TOKEN_TRANSFERS.includes(event.evm_log_name),
```

Comment:
WANTED_EVM_LOG_EVENTS_FOR_LISTING_TOKEN_TRANSFERS is not needed anymore

Reply:
Right. Removed.